pkg/ini/ini.go

```diff
@@ -183,11 +183,10 @@ func WakaHomeDir() (string, error) {
 	home, exists := os.LookupEnv("WAKATIME_HOME")
 	if exists && home != "" {
 		home, err := homedir.Expand(home)
-		if err != nil {
```

Comment:
I think the convention is to always check `if err != nil` and never `if err == nil`?

Reply:
Not always. The key is to return short cases first.